File: DataAccess/Models/Attachment.cs
namespace DataAccess.Models
{
    public class Attachment
    {
        public string? Company_Code { get; set; }

        public int AttachmentId { get; set; }

        public int CustomerId { get; set; }

        public int JobAddressId { get; set; }

        public string? FunctionTable { get; set; }

        public int? FunctionId { get; set; }

        public string? AttachmentType { get; set; }

        public string? AttachmentName { get; set; }

        public string? AttachmentBase64String { get; set; }

        public string? AttachmentPath { get; set; }
    }
}

File: AccutracWebsite/src/app/core/interfaces/customer/attachment.ts
import { SafeResourceUrl } from "@angular/platform-browser";

export interface Attachment {
  attachmentId?: number;
  company_Code: string;
  customerId?: string;
  jobAddressId?: number;
  functionTable?: string;
  functionId?: number;
  attachmentType: string;
  attachmentName: string;
  attachmentPath?: string;
  attachmentBase64String?: string;
  attachmentUrl?: SafeResourceUrl;
  attachmentByteArray?: any[]
}
